packages/web-components/docs/api-report.md

```diff
@@ -74,6 +75,8 @@ export interface AccordionItem extends StartEnd {
 // @public (undocumented)
 export const accordionItemDefinition: FASTElementDefinition<typeof AccordionItem>;
 
+// Warning: (ae-missing-release-tag) "AccordionItemExpandIconPosition" is part of the package's API, but it is missing a release tag (@alpha, @beta, @public, or @internal)

```

Comment:
There seem to be a lot of the `ae-missing-release-tag` will there be a follow up PR adding them?

Reply:
The majority of these will be resolved over time as components get reworked - an important focus area for each component upgrade is adding proper documentation.

Reply:
@radium-v I think this PR is all good, but would be good to get resolution on this to unblock the PR in its current state